src/Symfony/Component/AssetMapper/Command/ImportMapInstallCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\ImportMap\ImportMapManager;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Downloads each missing downloaded asset.
+ *
+ * @author Jonathan Scheiber <contact@jmsche.fr>
+ */
+#[AsCommand(name: 'importmap:install', description: 'Downloads each missing downloaded asset.')]
```

Comment:
```suggestion
#[AsCommand(name: 'importmap:install', description: 'Download all assets that should be downloaded.')]
```

Mostly, I'm trying to avoid the word "missing" - that makes it seem like the command is fixing a problem vs being part of a normal workflow.